### ISSUE TITLE
[Feature request/proposal]Transfer Pokemon by more accurate stats (Attack, HP, Stamina)

### ISSUE THREAD

**Issue by Mixone-FinallyHere** [jabbink/PokemonGoBot#980]
Feature request/proposal
----------------------------
**Description:**
When automating transfer we can only choose total IV and CP for delimiters of transfer.
How about adding the possibility of Attack or Stamina or Health as logic for the transfer instead or as well as?

Say you want combat pokemon:
- Set transfer_below_attack to 15 to only have the pokemon with 15 stat for attack
- Similarly for the other stats.

**Expected behavior:**
Being able to select pokemon that are better for either:
- Fighting hard                       => *Attack*
- Fighting long                       => *Life*
- Evolving to ridiculous CP     => *Stamina* (I think)

**Actual behavior:**
You can only choose between CP and IV and can get pokemon that aren't as good at what you wanted them to be good at.

**Version:**
Up to the author I guess.

**Operating System:**
Mine is Windows and Ubuntu but I guess the apple people can have it too....

**Java version:**
I drink tea rather

**Comment by pashorizer** [comment]
So you would like to have a Pokemon with 15A, but 1L and 1S? I don't think micromanaging the IV stats is a great idea, because in the end you would get the very best from sniping anyways and release/sort it by hand.

**Comment by Mixone-FinallyHere** [comment]
yes but say you want an attack team but dont care if they wont get much better CP
set CP to 1000 (for example)
set Attack to 15
set HP to 15
now this means that all you keep will have at least 1000 CP best attack and life at their CP but no matter what stamina

because if you look at IV
attack 15
hp 15
stamina 0
gives 30/45 giving IV of 67%

which means that by setting IV to 70 you will lose any of them that meet that criterion 

I could go on but I think the point is clear, the bot calculates them anyway, why not store them and use them? 
I'm not saying lose the IV transfer limit, just add the possibility of this being set instead of IV

might be bad for the unexperienced but once you want to start destroying gyms pokemon with lower stats can get through the IV block easily

**Comment by Casual-User** [comment]
I support this need – it would help a lot to have class specific criteria and – after that – the possibility to have less rigid criteria that would allow to produce purpose adequate PMs out of a greater group of keepers.
